src/Umbraco.Infrastructure/BackgroundJobs/Jobs/LongRunningOperationsCleanupJob.cs

```diff
@@ -0,0 +1,53 @@
+using Umbraco.Cms.Core.Persistence.Repositories;
+using Umbraco.Cms.Core.Scoping;
+
+namespace Umbraco.Cms.Infrastructure.BackgroundJobs.Jobs;
+
+/// <summary>
+/// Cleans up long-running operations that have exceeded a specified age.
+/// </summary>
+public class LongRunningOperationsCleanupJob : IRecurringBackgroundJob
+{
+    private readonly ICoreScopeProvider _scopeProvider;
+    private readonly ILongRunningOperationRepository _longRunningOperationRepository;
+    private readonly TimeProvider _timeProvider;
+
+    private readonly TimeSpan _deleteTime = TimeSpan.FromDays(1);
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LongRunningOperationsCleanupJob"/> class.
+    /// </summary>
+    /// <param name="scopeProvider">The scope provider for managing database transactions.</param>
+    /// <param name="longRunningOperationRepository">The repository for managing long-running operations.</param>
+    public LongRunningOperationsCleanupJob(
+        ICoreScopeProvider scopeProvider,
+        ILongRunningOperationRepository longRunningOperationRepository,
+        TimeProvider timeProvider)
+    {
+        _scopeProvider = scopeProvider;
+        _longRunningOperationRepository = longRunningOperationRepository;
+        _timeProvider = timeProvider;
+    }
+
+    /// <inheritdoc />
+    public event EventHandler? PeriodChanged
+    {
+        add { }
+        remove { }
+    }
+
+    /// <inheritdoc />
+    public TimeSpan Period => TimeSpan.FromMinutes(2);
```

Comment:
Perhaps we should have configuration for some of these options on when the job runs, and in particular how many days back to clean up.  See for example `CacheInstructionsPruningJob` and `HealthCheckNotifierJob`.

Reply:
Adjusted in https://github.com/umbraco/Umbraco-CMS/pull/19688/commits/7c572e5e16a7e647c5453792fe8747bcf6aabbe6 but not sure if any other changes are needed.